com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -566,6 +571,73 @@ private void OnDestroy()
             DisposeDriver();
         }
 
+#if MULTIPLAYER_TOOLS
+        private void ExtractNetworkMetrics()
+        {
+            if (NetworkManager.Singleton.IsServer)
+            {
+                var ngoConnectionIds = NetworkManager.Singleton.ConnectedClients.Keys;
+                foreach (var ngoConnectionId in ngoConnectionIds)
+                {
+                    //Connection 0 when host is self and bypass the transport so would be invalid, skip it.
+                    if (ngoConnectionId > 0 || (ngoConnectionId == 0 && !NetworkManager.Singleton.IsHost))
+                    {
+                        ExtractNetworkMetricsForClient(NetworkManager.Singleton.ClientIdToTransportId(ngoConnectionId));
+                    }
```

Comment:
Rather than explaining the logic in a comment, you could write something like the following:
```
var isConnectionToSelf = ngoConnectionId == 0 && NetworkManager.Singleton.IsHost;
if (isConnectionToSelf)
{
 continue;
}
ExtractNetworkMetricsForClient(NetworkManager.Singleton.ClientIdToTransportId(ngoConnectionId));
```

Reply:
This also reduced the complexity of the condition a bit.